lwutil.h

```diff
@@ -88,6 +88,22 @@ extern "C" {
 
 #define LWUTIL_CONCAT_BASE(x, y)                x ## y
 #define LWUTIL_CONCAT(s0, s1)                   LWC_CONCAT_BASE(s0, s1)
+/**
+ * \brief           Set a bit at specific position in a number, other bits are unchanged
+ * \param[in]       num: Number in which bit needs to be set
+ * \param[in]       pos: Position of bit to be set
+ * \return          number with bit at 'pos' set
+ */
+
+#define LWUTIL_SET_BIT(num, pos)				num = num | (1<<pos)
```

Comment:
This code is not safe, especially if num is an expression.

Reply:
Can you elaborate? I am not sure in which scenario num will be an expression? 
As i understand it we can only set/clear bits on variables